### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by danintel**
- Add answer on Avalon nodes
- Add Singleton, KME, and WPE to glossary
- Add workload ID and worker pool

Signed-off-by: danintel <daniel.anderson@intel.com>

### CODE REVIEW

File: docs/FAQ.rst [hunk 1]
```diff
@@ -526,6 +536,9 @@ KECCACK-256
     3", before Keccak was standardized to SHA-3 (FIPS-202) and Keccak has
     minor variations from SHA-3
 
+KME
+    Key Management Enclave Avalon worker. Compare to KME and WPE
```
Comment (EugeneYYY) [inline docs/FAQ.rst:540]: 1) It should be "Compare with _**Singleton**_ and WPE" 
2) I would add (before the phrase above) "KME is a part of the worker pool responsible for the key management. It has access to the worker's private keys and controls workorder execution by WPE"

File: docs/FAQ.rst [hunk 2]
```diff
@@ -722,6 +739,9 @@ Work order response
     A response generated by an Avalon worker to a work
     order. The response is a JSON string
 
+WPE
+    Work order Processor Enclave Avalon worker. Compare with KME and WPE
```
Comment (EugeneYYY) [inline docs/FAQ.rst:743]: It should be "Compare with KME and _**Singleton**_"
I would add (before the phrase above) "WPE is a part of the worker pool responsible for the workorder processing. It doesn't has access to the worker's private keys and relies on the KME to provide necessary context for the workorder processing."

File: docs/FAQ.rst [hunk 3]
```diff
@@ -722,6 +739,9 @@ Work order response
...
+
```
Comment (EugeneYYY) [inline docs/FAQ.rst:744]: I would add worker pool definition also

Comment (danintel) [reply docs/FAQ.rst:744]: Added. I also added workload ID.

**Comment by danintel** [comment]
Hi Eugene,
I made your suggested changes—please re-review. I also added definitions for workload ID and worker pool.

Dan

**Comment by EugeneYYY** [comment]
Done. Nice addition to the FAQ!



Regards,
Eugene

**Pull request merged**